### PR TITLE
feat(world): add transformer from v2 config output to v1 config output

### PR DESCRIPTION
**Pull request by alvrs**

### CODE REVIEW

File: packages/world/ts/config/v2/output.ts [hunk 1]
```diff
@@ -1,4 +1,47 @@
 import { Config as StoreConfig, Table } from "@latticexyz/store/config/v2";
+import { ModuleConfig } from "./input";
+
+export type SystemConfig = {
+  /** The full resource selector consists of namespace and name */
```
Comment (holic) [inline packages/world/ts/config/v2/output.ts:5]: is this an old comment? not sure what it's referring to

File: packages/world/ts/config/v2/output.ts [hunk 2]
```diff
@@ -1,4 +1,47 @@
...
+  readonly openAccess: boolean;
+  /** An array of addresses or system names that can access the system */
+  readonly accessList: string[];
+};
+
+export type SystemsConfig = { readonly [key: string]: SystemConfig };
+
+export type DeploymentConfig = {
+  /** The name of the World contract to deploy. If no name is provided, a vanilla World is deployed */
+  readonly worldContractName: string | undefined;
```
Comment (holic) [inline packages/world/ts/config/v2/output.ts:25]: thoughts on naming this something like `customWorldContract`?

File: packages/world/ts/config/v2/output.ts [hunk 3]
```diff
@@ -1,4 +1,47 @@
 import { Config as StoreConfig, Table } from "@latticexyz/store/config/v2";
+import { ModuleConfig } from "./input";
+
+export type SystemConfig = {
+  /** The full resource selector consists of namespace and name */
+  readonly name: string | undefined;
```
Comment (holic) [inline packages/world/ts/config/v2/output.ts:6]: when would this be undefined?

File: packages/world/ts/config/v2/output.ts [hunk 4]
```diff
@@ -8,4 +51,13 @@ export type Config = StoreConfig & {
       };
     };
   };
+  readonly systems: SystemsConfig;
+  /** Systems to exclude from automatic deployment */
```
Comment (holic) [inline packages/world/ts/config/v2/output.ts:55]: system names?

**Review by holic — APPROVED** [review]
nothing blocking but would like to be mindful of naming+shapes of output types if we expect downstream users of em

File: packages/world/ts/config/v2/output.ts (hunk 4 above)
Comment (alvrs) [reply packages/world/ts/config/v2/output.ts:55]: are you referring to the comment for `excludeSystems`?

Comment (holic) [reply packages/world/ts/config/v2/output.ts:55]: yeah, I think this is system contract names to exclude?

File: packages/world/ts/config/v2/output.ts (hunk 1 above)
Comment (alvrs) [reply packages/world/ts/config/v2/output.ts:5]: yeah was an old comment, updated

File: packages/world/ts/config/v2/output.ts (hunk 3 above)
Comment (alvrs) [reply packages/world/ts/config/v2/output.ts:6]: good catch, never undefined

**Pull request merged**